docs/src/content/docs/reference/configuration.md

```diff
@@ -5,6 +5,22 @@ description: An overview of all the configuration options Starlight supports.
 
 ## Configure the `starlight` integration
 
+Starlight is an [Astro integration](https://docs.astro.build/en/guides/integrations-guide/), configured in `astro.config.mjs`.
```

Comment:
I wouldn't overwhelm the user with that page of docs, or feeling the need to understand much about Astro, integrations, etc. Instead, I might reframe to something like this:

```suggestion
Starlight is built on top the [Astro](https://astro.build) web framework. You can configure your project inside the Astro `astro.config.mjs` configuration file, like this:
```